test/e2e_broker/conformance/control_plane_conformance_test.go

```diff
@@ -20,29 +20,26 @@
 package conformance
 
 import (
-	"fmt"
 	"testing"
 
-	conformance "knative.dev/eventing/test/conformance/helpers"
-	testlib "knative.dev/eventing/test/lib"
-	"knative.dev/eventing/test/lib/resources"
+	eventing "knative.dev/eventing/pkg/apis/eventing/v1"
 
 	pkgtesting "knative.dev/eventing-kafka-broker/test/pkg"
-	"knative.dev/eventing-kafka-broker/test/pkg/broker"
+	brokertest "knative.dev/eventing-kafka-broker/test/pkg/broker"
+	conformance "knative.dev/eventing/test/conformance/helpers"
+	testlib "knative.dev/eventing/test/lib"
 )
 
 func brokerCreator(client *testlib.Client, name string) {
-
-	class, err := broker.GetKafkaClassFromEnv()
```

Comment:
the `GetKafkaClassFromEnv` is now inlined to `CreatorWithOption`, right? 

I am wondering why to was before explicit on a few tests here 
(Not saying we need to factor out this change, just curious)

Reply:
There's no big reason other than me not seeing there's a better, unified and reusable way.